includes/class-sensei-frontend.php

```diff
@@ -959,6 +963,7 @@ public function sensei_complete_lesson_button() {
 		if ( false === Sensei()->lesson->lesson_has_quiz_with_questions_and_pass_required( $lesson_id ) ) {
 
 			wp_enqueue_script( 'sensei-stop-double-submission' );
+			wp_enqueue_script( 'sensei-disable-complete-lesson-button' );
```

Comment:
Sorry if I missed something, but shouldn't it be `sensei-course-video-blocks-youtube`?

Reply:
Auch, actually, I have to remove this statement. Thanks.

Reply:
Removed here: https://github.com/Automattic/sensei/pull/4546/commits/6f6e6048e03ad8ad103d6fb8a84b0442ea47a5cd